selfdrive/car/gm/interface.py

```diff
@@ -161,6 +162,8 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None, expe
       ret.centerToFront = ret.wheelbase * 0.5
 
     elif candidate == CAR.ESCALADE_ESV:
+      ret.radarOffCan = False  # 0x2cb not in Escalade fingerprint even though it has radar. Likely incomplete fingerprint.
+      ret.dashcamOnly = False  # TODO: Remove this special logic if Escalade is re-fingerprinted
```

Comment:
you should be able to eventually get rid of this, I can update the fingerprint in another PR if it's incorrect